### DISCUSSION TITLE
Habrá video explicativo de las prácticas?

### DISCUSSION THREAD

**Discussion by DanielOe21** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#22] [Q&A]
Si no mal recuerdo, en el live del miércoles casi al final, se comentó que se subiría un video para explicar las prácticas,

¿Alguien sabe si hay algún cambio con esto? O sería cuestión de esperar?

**Comment by soygussil** [comment]
También tengo esa grandísima duda 😅😅😅

**Reply by DanielOe21** [reply]
Por lo menos yo estoy medio perdido 🥲 o sea entendí el caso práctico pero no entendí los entregables 😜

**Reply by xian145** [reply]
en el video del miércoles se abordo un poco sobre estas cosas, dio con ejemplos lo que leímos en introducción, pero siendo sincero, sigo bastante perdido, siento que lo que pide en las practicas es bastante técnico, al igual que ustedes, estaba esperando el video, de igual manera tenemos hasta el viernes para entregar

**Comment by hrrRafael2** [comment]
que dudas tienen en la practica?

**Comment by PaulHRios** [comment]
Pues podemos esperar al lunes y preguntar, no entendí muy bien la parte de requerimientos, pero puedes ir pensando en las siguientes partes de la práctica. Por ejemplo hacer un perfil de la persona que será cliente del servicio de abogabot, su edad, su nivel educativo, etc. Algo similar a lo que hace Google con nosotros. Visitando esta página https://adssettings.google.com/authenticated puedes ver lo que Google sabe o cree saber de ti y te puede ser útil para saber que información le interesa a las empresas sobre ti.
También tienes que ir haciendo tu wireframe que sería el esqueleto o la forma en la que estará distribuida y luego en su UI con la paleta de colores y todo eso.

No se preocupen esta práctica tenemos hasta el 25 de febrero para entregarla.

**Comment by GloriaCoralCerecedo** [comment]
Hola, yo digo que el lunes el MC a lo mejor resuelve dudas que tengamos. Mientras podríamos revisar todos y apuntar nuestras dudas.
